hack/development/Dockerfile.sidecar

```diff
@@ -15,7 +15,7 @@ RUN apt-get update \
 COPY hack/docker/rclone.gpg /root/rclone.gpg
 RUN gpg --import /root/rclone.gpg
 
-ENV RCLONE_VERSION=1.53.1
+ENV RCLONE_VERSION=1.53.4
```

Comment:
Thx, Why not just upgrade to 1.57.0?

Reply:
This is because most people will merge in a patch, even if they refuse to do so for a minor version upgrade.
Since you seem to be a person who welcomes minor versions, I will do so.